### PR TITLE
Upgrade dependencies to the latest releases & pin compiler version

### PR DESCRIPTION
**Pull request by Geod24**
Because master is currently broken.

### CODE REVIEW

**Comment by Geod24** [comment]
Damn it @dlang-bot

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 language: d
+
 d:
-  - dmd
-  - ldc
-sudo: false
+  - dmd-2.090.1
+  - ldc-1.20.0
```
Comment (wilzbach) [inline .travis.yml:5]: This will get outdated very soon.

Comment (Geod24) [reply .travis.yml:5]: We need a new Vibe.d release for that.

Comment (wilzbach) [reply .travis.yml:5]: Ah. I didn't realize this. Sorry!

Comment (Geod24) [reply .travis.yml:5]: For reference, this is the required PR to get things to build with Vibe.d and > 2.090 FE again: https://github.com/vibe-d/vibe.d/pull/2439

Comment (wilzbach) [reply .travis.yml:5]: I guess we could have used the beta: https://github.com/vibe-d/vibe.d/releases/tag/v0.9.0-beta.1

**Pull request merged**